notify/notify.go

```diff
@@ -407,27 +440,5 @@ func (n *InhibitNotifier) Notify(ctx context.Context, alerts ...*types.Alert) er
 		}
 	}
 
-	return n.notifier.Notify(ctx, filtered...)
-}
-
-// LogNotifier logs the alerts to be notified about. It forwards to another Notifier
-// afterwards, if any is provided.
-type LogNotifier struct {
```

Comment:
Removed because no longer used?

Totally fair, this shouldn't exist anymore. If we want to do any debug logging between stages or at the very end, that should be done directly.